extensions/data-workspace/src/common/workspaceTreeDataProvider.ts

```diff
@@ -115,6 +115,14 @@ export class WorkspaceTreeDataProvider implements vscode.TreeDataProvider<Worksp
 	}
 
 	handleDrag(treeItems: readonly WorkspaceTreeItem[], dataTransfer: vscode.DataTransfer): void | Thenable<void> {
+		// don't do anything if trying to drag the project node since it isn't supported and there's an error when trying to
+		// convert the project node to JSON https://github.com/microsoft/azuredatastudio/issues/22529
```

Comment:
Given that this fix is _closing_ this bug rather than, say, a work-around or something that keeps it present, we probs don't need a link to the soon-to-be-closed item.

Reply:
I was more putting it there so I don't forget in the future, but I can look back at the PR and see the linked issue :)